keyboards/custommk/elysian/keymaps/default/keymap.c

```diff
@@ -0,0 +1,19 @@
+// Copyright 2024 customMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_ESC, MO(1),
```

Comment:
`MO(1)` here would go out of bounds.

Reply:
Changed default keycode assignment to avoid going out of bounds, thank you!